lib/modules/platform/gerrit/index.spec.ts

```diff
@@ -187,28 +187,6 @@ describe('modules/platform/gerrit/index', () => {
       gerrit.writeToConfig({ labels: {} });
     });
 
-    it('updatePr() - auto approve enabled', async () => {
```

Comment:
why so many removed tests? I see missing coverage

Reply:
Because `approveChange` no longer runs in `updatePr` function, this test is now obsolete. Also, `wasApprovedBy` function no longer exists because it is no longer needed, thus I removed tests of it.

Reply:
I will fix coverage. I had not realized it had decreased. Thank you.